R-package/NEWS.md

```diff
@@ -1,3 +1,9 @@
+## RGF 1.0.5.2
```

Comment:
Thank you @mlampros very much!

One note: since `1.0.5.1` has never been released, maybe squash all changes into `1.0.6` version and release it later (after uploading `rgf_python` with warm-start functionality to PyPI)?

Reply:
Yeah, cool! Agree with you.